packages/node_modules/@ciscospark/widget-recents/src/reducer.js

```diff
@@ -30,6 +30,7 @@ const Status = Record({
 
 const RecentsWidget = Record({
   incomingCall: null,
+  keyword: {},
```

Comment:
Can you make this either `null` or `''` since it is just a string going in here and not an object?